.github/hacktoberfest.md

```diff
@@ -0,0 +1,24 @@
+# Hacktoberfest
+
+During October 2022, Astro’s docs repo will accept your PRs for [Hacktoberfest](https://hacktoberfest.com/). Here’s how to take part!
```

Comment:
```suggestion
During October 2022, Astro’s docs repo will accept your PRs for [Hacktoberfest](https://hacktoberfest.com/)! Here’s how to take part:
```
Hurricane can't stop the nits... :purple_heart:

Reply:
Such resilient little nits…